pyrit/orchestrator/scoring_orchestrator.py

```diff
@@ -91,7 +88,11 @@ async def score_prompts_by_request_id_async(
 
         request_pieces = self._remove_duplicates(request_pieces)
 
-        return await scorer.score_prompts_batch_async(request_responses=request_pieces, batch_size=self._batch_size)
+        return await scorer.score_prompts_with_tasks_batch_async(
+            request_responses=request_pieces,
+            batch_size=self._batch_size,
+            tasks=[task for _ in range(len(request_pieces))],
```

Comment:
Nit: could avoid list comprehension
```suggestion
            tasks=[task] * len(request_pieces)
```